.github/workflows/agent-claude.yml

```
@@ -0,0 +1,101 @@
+jobs:
+  claude:
+    # Allow specific bots (@copilot[bot], @devin[bot], @coderabbitai[bot]) and all humans to trigger Claude
+    if: |
+      (github.event_name == 'issue_comment' &&
+       github.event.issue.pull_request &&
+       contains(github.event.comment.body, '@claude') &&
+       (github.event.comment.user.type != 'Bot' ||
+        contains(fromJSON('["copilot[bot]", "devin[bot]", "coderabbitai[bot], cursor[bot]"]'), github.event.comment.user.login))) ||
+      (github.event_name == 'pull_request_review_comment' &&
+       contains(github.event.comment.body, '@claude') &&
+       (github.event.comment.user.type != 'Bot' ||
```

actionlint (1.7.7)

broken JSON string is passed to fromJSON() at offset 53: invalid character 'c' looking for beginning of value

(expression)